### PR TITLE
add task solution MR

### PR DESCRIPTION
**Pull request by RuslanMelnykFE**
- [DEMO LINK](https://RuslanMelnykFE.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://RuslanMelnykFE.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,46 @@
...
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover"
+            >laptops & computers</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:36]:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,46 @@
...
+        <img
+          src="./images/logo.png"
+          alt="Logo MOYO"
+          class="header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
```
Comment (volodymyr-soltys97) [inline src/index.html:23]: let's make links more informative
```suggestion
            <a href="#apple" class="nav__link is-active">apple</a>
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,94 @@
...
+  color: #000;
+}
+
+.nav__link:hover {
+  color: var(--active-color-link);
+}
+
+.nav__link:active {
+  color: var(--active-color-link);
+}
```
Comment (volodymyr-soltys97) [inline src/style.css:77]: Redundant
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,94 @@
...
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
```
Comment (volodymyr-soltys97) [inline src/style.css:60]: Move box-sizing to * selector

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,94 @@
...
+  height: 100%;
+  min-height: 40px;
+  font-size: 12px;
+  font-weight: 500;
```
Comment (volodymyr-soltys97) [inline src/style.css:64]: The default value, no need to write it
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Remove margin left for the first child
image

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,94 @@
...
+  line-height: 0;
+}
+
+.header__logo {
+  width: 100%;
+  height: 100%;
+}
+
+/* .nav {
+  padding: 10px 0;
```
Comment (mykhalenych) [inline src/style.css:40]: Screenshot 2023-08-10 at 18 01 51

did you redeploy?
I don't see changes, also delete comments

File: src/style.css (hunk 3 above)
Comment (mykhalenych) [reply src/style.css:77]: https://github.com/mate-academy/layout_moyo-header/pull/3683/files#r1289797204
didn't fix it

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,86 @@
...
+  position: relative;
+  color: var(--active-color-link);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  left: 50%;
+  bottom: 0;
+  transform: translateX(-50%);
```
Comment (volodymyr-soltys97) [inline src/style.css:81]: Redundant
```suggestion
  bottom: 0;
```

File: src/style.css [hunk 8]
```diff
@@ -1,3 +1,86 @@
...
+.header__logo {
+  width: 100%;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  justify-content: flex-end;
```
Comment (volodymyr-soltys97) [inline src/style.css:43]: Redundant
```suggestion
```

**Review by volodymyr-soltys97 — APPROVED** [review]

**Pull request closed without merging**